plugins/com.cubrid.common.core/src/com/cubrid/common/core/util/QuerySyntax.java

```diff
@@ -2054,6 +2054,25 @@ public static String escapeKeyword(String text) {
             return "";
         }
 
+        String[] parts = text.split("\\.");
+
+        if (parts.length == 2) {
+            StringBuilder escapedText = new StringBuilder();
+            for (int i = 0; i < parts.length; i++) {
+                String part = parts[i];
+                boolean isValidate = checkIdentifier(part);
+                if (isValidate) {
+                    escapedText.append(part);
+                } else {
+                    escapedText.append("[" + part + "]");
```

Comment:
```suggestion
                    escapedText.append("[").append(part).append("]");
```
append 체이닝을 사용하는 것이 더 효율적일 것 같습니다!

Reply:
@Srltas 
수정하였습니다.